utils/callApi.js

```diff
@@ -19,6 +19,7 @@ const callApi = async ({ method, endpoint, body }, inputs) => {
 
   if (response.status !== 200) {
     logWarning(`Please ensure that Github App is installed ${GITHUB_APP_URL}`)
+    return response
```

Comment:
Wouldn't this tell the user that every error is about the app not being installed? Not sure if the server returns useful info about errors but if it does, I'd be good to log them, I think.

Reply:
I guess the `setFailed` will give the user feedback per action failure, right? we can logError as well but I think it may be a bit redundant

Reply:
You're right @ja-bravo that this warning we're logging is really generic, that's certainly something to improve for the future but not really in scope for this PR.